example/validator-azure-certs/tls.crt

```diff
@@ -0,0 +1,23 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
It doesn't harm, but this cert contains "Provider AWS" which might be confusing.